frontend/src/components/SignIn/SignIn.js

```diff
@@ -42,6 +52,27 @@ class SignIn extends Component {
       });
   };
 
+
+  handleSuspendedUser = () => {
```

Comment:
I think we'll actually want to handle this on the backend, so that Devise doesn't allow a user to log in. Maybe we could add an `active_for_authentication?` method to the User model in Rails, which returns false if the user is suspended, like in https://stackoverflow.com/questions/6004216/devise-how-do-i-forbid-certain-users-from-signing-in